deal.II-toolchain/packages/superlu_dist.package

```diff
@@ -1,25 +1,29 @@
 ################################################################################
-## SUPERLU_DIST                                                               ##
-################################################################################
+# SUPERLU_DIST
 
-VERSION=5.1.2
-CHECKSUM=0a99bbb5f6c26a9c1d7ab17ea35d4e6f
+################################################################################
+VERSION=7.0.0
+CHECKSUM=skip
```

Comment:
```Suggestion
VERSION=7.0.0
CHECKSUM=b6f099f722b64c2dc647b3bc7eb43b74d6e18520e054ef32de02c74c11ecafc4
CHECKSUM="${CHECKSUM} 5161fb0fdc54833fecbb0f7d03ed90a9e021c7c8"
CHECKSUM="${CHECKSUM} a3d0f832820ca4dcab6e1fc0c2b5eb1c"
```

Reply:
v7 doesn't link correctly; superlu_dist is not of interest here